sol/main.c

```diff
@@ -508,7 +508,15 @@ run_sol(struct net_config *net_conf, struct sol_config *sol_conf)
 		goto out;
 	}
 
-	ret = init_mailbox("sol_reqs", 2 * sol_conf->pri_req_max_len,
```

Comment:
Replace `2 * sol_conf->pri_req_max_len` with `rte_align32pow2(2 * sol_conf->pri_req_max_len)` instead of the code below. Remember to drop the new variable `mailbox_max_entries_exp`.